docs/source/json-api/production-setup.rst

```diff
@@ -6,397 +6,12 @@ Production Setup
-
-A timer. Measures latency (in milliseconds) for processing of a query GET request.
-
-``daml.http_json_api.query_matching_timing``
---------------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a query POST request.
-
-``daml.http_json_api.fetch_timing``
------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a fetch request.
-
-``daml.http_json_api.get_party_timing``
----------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a get party/parties request.
-
-``daml.http_json_api.allocate_party_timing``
---------------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a party management request.
-
-``daml.http_json_api.download_package_timing``
-----------------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a package download request.
-
-``daml.http_json_api.upload_package_timing``
---------------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing of a package upload request.
-
-``daml.http_json_api.incoming_json_parsing_and_validation_timing``
-------------------------------------------------------------------
-
-A timer. Measures latency (in milliseconds) for parsing and decoding of an incoming json payload
-
-``daml.http_json_api.response_creation_timing``
--------------------------------------------------------
-
-A timer. Measures latency (in milliseconds) for construction of the response json payload.
-
-``daml.http_json_api.db_find_by_contract_key_timing``
------------------------------------------------------
-
-A timer. Measures latency (in milliseconds) of the find by contract key database operation.
-
-``daml.http_json_api.db_find_by_contract_id_timing``
-----------------------------------------------------
-
-A timer. Measures latency (in milliseconds) of the find by contract id database operation.
-
-``daml.http_json_api.command_submission_ledger_timing``
--------------------------------------------------------
-
-A timer. Measures latency (in milliseconds) for processing the command submission requests on the ledger.
-
-``daml.http_json_api.http_request_throughput``
-----------------------------------------------
-
-A meter. Number of http requests
-
-``daml.http_json_api.websocket_request_count``
-----------------------------------------------
-
-A Counter. Count of active websocket connections
-
-``daml.http_json_api.command_submission_throughput``
-----------------------------------------------------
-
-A meter. Number of command submissions
-
-``daml.http_json_api.upload_packages_throughput``
--------------------------------------------------
-
-A meter. Number of package uploads
-
-``daml.http_json_api.allocation_party_throughput``
---------------------------------------------------
-
-A meter. Number of party allocations
+.. toctree::
```

Comment:
No tocs in this repo please, add them to [docs.daml.com](https://github.com/digital-asset/docs.daml.com).

Reply:
What does this mean, when json-api/index has one?

Reply:
eb464700025c2232c5316b845b4ab24476288964

Reply:
digital-asset/docs.daml.com#55